src/game_logic/game_world.hpp

```diff
@@ -115,13 +115,13 @@ class GameWorld : public entityx::Receiver<GameWorld>
   friend class rigel::GameRunner;
 
 private:
-  struct ViewportParams
+  struct viewportParams
```

Comment:
Structs always start with an uppercase letter in the RigelEngine code base. So the correct name here is `ViewportParams`.